libcudacxx/include/cuda/std/detail/libcxx/include/__functional_03

```diff
@@ -1,1595 +0,0 @@
-// -*- C++ -*-
-//===----------------------------------------------------------------------===//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
This is missing to remove the files from libcudacxx/include/cuda/std/detail/libcxx/include/CMakeLists.txt